sub-packages/bionemo-geneformer/src/bionemo/geneformer/run/recipes.py

```diff
@@ -0,0 +1,610 @@
+
+    return MainConfig(
+        data_config=data_config,
+        parallel_config=parallel_config,
+        training_config=training_config,
+        bionemo_model_config=geneformer_config,
+        optim_config=optim_config,
+        experiment_config=experiment_config,
+    )
+
+
+def pretrain_tiny_test_recipe(args) -> MainConfig[ExposedGeneformerPretrainConfig, GeneformerPretrainingDataConfig]:
+    """Recipe for pretraining a tiny model. Used in testing."""
+    data_path = args.data_path
+    result_dir = args.result_dir
+
+    parallel_config = ParallelConfig(
+        tensor_model_parallel_size=1, pipeline_model_parallel_size=1, num_devices=1, accumulate_grad_batches=2
+    )
+    training_config = TrainingConfig(
+        max_steps=10, limit_val_batches=2, val_check_interval=2, precision="bf16-mixed", accelerator="gpu"
+    )
+    data_config = GeneformerPretrainingDataConfig(
+        seq_length=128,
+        micro_batch_size=2,
+        num_dataset_workers=0,
+        data_dir=data_path,
+    )
+    experiment_config = ExperimentConfig(
+        save_every_n_steps=training_config.val_check_interval,
+        result_dir=result_dir,
+        experiment_name="test-experiment",
+        restore_from_checkpoint_path=None,
+        save_last_checkpoint=True,
+        metric_to_monitor_for_checkpoints="reduced_train_loss",
+        save_top_k=2,
+        create_tensorboard_logger=False,
+    )
+
+    optim_config = OptimizerSchedulerConfig()
+    geneformer_config = geneformer_tiny_config(
+        seq_length=data_config.seq_length, initial_ckpt_path=args.initial_ckpt_path
+    )
+
+    return MainConfig(
+        data_config=data_config,
+        parallel_config=parallel_config,
+        training_config=training_config,
+        bionemo_model_config=geneformer_config,
+        optim_config=optim_config,
+        experiment_config=experiment_config,
+    )
+
+
+def geneformer_10m_pretrain_recipe(
+    args,
+) -> MainConfig[ExposedGeneformerPretrainConfig, GeneformerPretrainingDataConfig]:
+    """Recipe for pretraining the 10m model."""
+    data_config: GeneformerPretrainingDataConfig = geneformer_data_recipe(data_dir=args.data_path)
+    parallel_config = simple_parallel_recipe()
+    training_config = geneformer_base_training_config()
+    bionemo_model_config = geneformer_10m_model_config(initial_ckpt_path=args.initial_ckpt_path)
+    optim_config = geneformer_base_optimizer_scheduler_config()
+    experiment_config = geneformer_10m_experiment_config(result_dir=args.result_dir)
+    wandb_config = geneformer_10m_wandb_config()
+    main_config = MainConfig[ExposedGeneformerPretrainConfig, GeneformerPretrainingDataConfig](
+        data_config=data_config,
+        parallel_config=parallel_config,
+        training_config=training_config,
+        bionemo_model_config=bionemo_model_config,
+        optim_config=optim_config,
+        experiment_config=experiment_config,
+        wandb_config=wandb_config,
+    )
+    return main_config
```

Comment:
@jstjohn does this look accurate?

Reply:
Yes other than `get_attention_mask_from_fusion=False,` which should be True. There is a small performance boost to have it set True.